doc/chapters/api/index.rst

```diff
@@ -3,131 +3,230 @@
 API
 ***
 
-This chapter documents the D-Bus API for interacting with the
-SoftwareContainerAgent. This is mostly useful for integrators.
+This chapter documents the D-Bus API for interacting with the SoftwareContainerAgent. This is mostly
+useful for integrators, or anyone writing a launcher for running apps inside SoftwareContainer
+containers.
 
 .. _dbus-api:
 
 D-Bus API
 =========
 
-D-Bus API is an IPC interface to call SoftwareContainer agent methods. The API
-provides following object path and interface.
+The D-Bus API is an interface to call SoftwareContainer agent methods. The API provides following
```

Comment:
For consistency write either "SoftwareContainerAgent" or "SoftwareContainer agent".